### PR TITLE
22 - Add Pay someone implementation

### PR DESCRIPTION
**Pull request by juliano60**
@michowski @ZacTredger Could you please review?

Thanks and Regards,
Jules

### CODE REVIEW

**Comment by ZacTredger** [comment]
Hey @juliano60 
Seems like the commit from `81 - Add default current account feature` is included in this PR as well. But I see you're still updating the branch, so maybe you know. Will you shout when you're ready for the review?
Z

**Comment by juliano60** [comment]
@ZacTredger I was tidying up the documentation. I thought about doing it in a separate PR but there wasn't too much to update. Feel free to review now, am done tweaking it. And correct, this PR uses the default account (item 81) as its base.

Kindest Regards,
Jules

**Comment by juliano60** [comment]
@ZacTredger Actually I ought to resolve the conflicts. Just a sec.

Kindest Regards,
Jules

File: docs/protocol.md [hunk 1]
```diff
@@ -121,6 +204,8 @@
   </tr>
 </table>
 
+<h2>Transactions</h2>
```
Comment (michowski) [inline docs/protocol.md:207]: Good idea!

**Review by ZacTredger — APPROVED** [review]
I think this looks great! Nice work
It's obviously very possible at the moment to accidentally pay the wrong person, but that's another issue for another time... (Perhaps a `contacts` feature?)

**Comment by juliano60** [comment]
@ZacTredger ,
Cheers buddy and correct we are using the customer's name as the customer's ID. The customer's ID could be derived from the name as well as other things (for instance a unique number generator). Something for the future.

Kindest Regards,
Jules

**Pull request merged**